experimental/learn_descriptors/four_seasons_parser_test.cc

```diff
@@ -16,13 +16,16 @@
 #include "GeographicLib/LocalCartesian.hpp"
 #include "absl/strings/str_split.h"
 #include "absl/strings/strip.h"
+#include "common/check.hh"
 #include "common/liegroups/se3.hh"
 #include "gtest/gtest.h"
 #include "nmea/message/gga.hpp"
 #include "nmea/message/rmc.hpp"
 #include "nmea/object/date.hpp"
 #include "nmea/sentence.hpp"
 
+#define CAM_HZ 30.0
```

Comment:
seems a bit unfortunate that this needs to be defined in a few places. Consider making it a `static constexpr` member variable on the parser.